packages/fxa-settings/src/components/ConnectAnotherDevicePromo/storeImageLoader.tsx

```diff
@@ -0,0 +1,145 @@
+import daApple from '../../assets/images/apple_app_store_button/da.svg';
+import deApple from '../../assets/images/apple_app_store_button/de.svg';
+import enApple from '../../assets/images/apple_app_store_button/en.svg';
+import esApple from '../../assets/images/apple_app_store_button/es.svg';
+import etApple from '../../assets/images/apple_app_store_button/et.svg';
+import frApple from '../../assets/images/apple_app_store_button/fr.svg';
+import heApple from '../../assets/images/apple_app_store_button/he.svg';
+import huApple from '../../assets/images/apple_app_store_button/hu.svg';
+import idApple from '../../assets/images/apple_app_store_button/id.svg';
+import itApple from '../../assets/images/apple_app_store_button/it.svg';
+import jaApple from '../../assets/images/apple_app_store_button/ja.svg';
+import koApple from '../../assets/images/apple_app_store_button/ko.svg';
+import ltApple from '../../assets/images/apple_app_store_button/lt.svg';
+import nbNOApple from '../../assets/images/apple_app_store_button/nb-NO.svg';
+import nlApple from '../../assets/images/apple_app_store_button/nl.svg';
+import plApple from '../../assets/images/apple_app_store_button/pl.svg';
+import ptBRApple from '../../assets/images/apple_app_store_button/pt-BR.svg';
+import ptApple from '../../assets/images/apple_app_store_button/pt.svg';
+import ruApple from '../../assets/images/apple_app_store_button/ru.svg';
+import skApple from '../../assets/images/apple_app_store_button/sk.svg';
+import slApple from '../../assets/images/apple_app_store_button/sl.svg';
+import svSEApple from '../../assets/images/apple_app_store_button/sv-SE.svg';
+import trApple from '../../assets/images/apple_app_store_button/tr.svg';
+import zhCNApple from '../../assets/images/apple_app_store_button/zh-CN.svg';
+import zhTWApple from '../../assets/images/apple_app_store_button/zh-TW.svg';
+
+import caGoogle from '../../assets/images/google_play_store_button/ca.png';
```

Comment:
Gaaaah do we really not have these google play images as SVGs anywhere? 😭 It looks like the English version is currently an SVG. Honestly, I would ask around to see if someone knows of some folder somewhere where these exist as translated SVGs. We prefer to [inline our SVGs](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/README.md#working-with-svgs) where we can to reduce the number of network requests, but my bigger problem with this is that unfortunately changing this from an SVG to a `.png` makes this blurry on my retina display, especially when you see it next to the nice crisp apple store SVG:

<img width="389" alt="image" src="https://user-images.githubusercontent.com/13018240/144679627-eaa4f672-a785-45cb-bd7e-7bb8b0d25a15.png">

You did also commit the `@2x` png versions, but to utilize those, we should be using the [`Picture` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) to render the correct version for us, meaning we'd need some slight refactoring _and_ you'd need to also import all of the `@2x` versions into this file. I'm really hoping we've got those SVGs somewhere...

Reply:
@LZoog I had a look around and we probably could find SVG versions somewhere, but I'm wondering if that's the right course of action.

The [Google badge generator](https://play.google.com/intl/en_us/badges/) only provides PNGs, and looking at the [Firefox homepage](https://www.mozilla.org/en-US/firefox/new/), it looks like they use the Google PNG there as well.

I did find these [unofficial SVGs](https://github.com/steverichey/google-play-badge-svg/tree/master/img), but I did notice the repo was last updated 4 years ago.

Reply:
I personally think using the SVGs is the better approach so we don't have to bother with `@2x` PNG images to account for retina displays and for consistency with all of our other images. I wonder if you can find something to help you compare PNG versions with the SVG versions because yeah... I personally think it's fine to use the unofficial SVGs you found, but it'd be most ideal to compare the PNGs with them to make sure the text on them looks similar. We could also ask to see if we have these officially as SVGs somewhere.

Reply:
As suggested by the team, I was able to convert the Google badges `EPS` files to `SVG`. 🎉